apps/src/turtle/turtle.js

```diff
@@ -1412,8 +1417,10 @@ Artist.prototype.checkAnswer = function () {
 
   // Test whether the current level is a free play level, or the level has
   // been completed
```

Comment:
probably worth updating this comment to reflect what this rather complicated boolean actually represents